weblate/static/loader-bootstrap.js

```diff
@@ -1525,16 +1525,50 @@ $(function () {
     $('.search-group li a').click(function () {
         var $this = $(this);
         var $button = $this.closest('.input-group-btn').find('button');
-        $button.data('field', $this.data('field'));
-        $button.find('span.search-label').text($this.text());
+        $button.attr('data-field', $this.data('field'));
+        $button.find('span.search-label').not('#query-sort-toggle span').text($this.text());
+        if ($this.data('sort')) {
+            $('#id_sort_by').val($this.data('sort'));
+            if ($('.pagination').length) {
```

Comment:
This is probably not the best way to decide we're on search results as there can be no pagination on sitewide search if all matches fit into single page.

Reply:
Umm. Okay. Since it's only in `search.html` and `translate.html` I can probably add a class in those two.

Reply:
That's probably the safest approach as the content of these pages can be changed and it's probably better not to rely on that as it will break without noticing later.

Reply:
Should be fixed.